.gitmodules

```diff
@@ -0,0 +1,24 @@
+[submodule "sc-flexy-datapoint-lib"]
+	path = sc-flexy-datapoint-lib
+	url = https://github.com/hms-networks/sc-flexy-datapoint-lib.git
+[submodule "flexy-azure-connector-generic"]
+	path = flexy-azure-connector-generic
+	url = https://github.com/hms-networks/flexy-azure-connector-generic.git
+[submodule "sc-flexy-historical-data-queue-lib"]
+	path = sc-flexy-historical-data-queue-lib
+	url = https://github.com/hms-networks/sc-flexy-historical-data-queue-lib.git
+[submodule "sc-flexy-string-lib"]
+	path = sc-flexy-string-lib
+	url = https://github.com/hms-networks/sc-flexy-string-lib.git
+[submodule "sc-flexy-tag-info-lib"]
+	path = sc-flexy-tag-info-lib
+	url = https://github.com/hms-networks/sc-flexy-tag-info-lib.git
+[submodule "sc-flexy-file-util-lib"]
+	path = sc-flexy-file-util-lib
+	url = https://github.com/hms-networks/sc-flexy-file-util-lib.git
+[submodule "sc-flexy-logger-lib"]
+	path = sc-flexy-logger-lib
+	url = https://github.com/hms-networks/sc-flexy-logger-lib.git
+[submodule "sc-flexy-json-lib"]
+	path = sc-flexy-json-lib
+	url = https://github.com/hms-networks/sc-flexy-json-lib.git
```

Comment:
The actual modules are not included in this commit. The modules themselves should also be committed/pushed to Git, otherwise they wont work on a fresh checkout. See the commit from the Azure connector here: https://github.com/hms-networks/flexy-azure-connector/commit/f2a66da801032b4edda7181c2d4cad9f1859b526. https://github.com/hms-networks/flexy-azure-connector/tree/master/lib

Reply:
yeah I wasn't sure on if we should include them or not. I probably should of just checked that before

Reply:
yeah without them, when you call git submodule update --init --remote --recursive, nothing happens.